backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java

```diff
@@ -1,31 +1,89 @@
 package dk.sdu.controllab.assembly.service;
 
+import com.google.gson.JsonObject;
 import dk.sdu.controllab.assembly.model.AssemblyRepository;
 import dk.sdu.controllab.common.Item;
 import dk.sdu.controllab.commonassembly.GetStatusResponse;
 import dk.sdu.controllab.commonassembly.IAssemblyService;
+import jakarta.annotation.Resource;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+import org.eclipse.paho.client.mqttv3.*;
+
+import java.util.Arrays;
+import java.util.UUID;
+import com.google.gson.Gson;
 
 @Service
 public class AssemblyServiceImpl implements IAssemblyService {
+    @Autowired
     private AssemblyRepository assemblyRepository;
-    //private MqttClient mqttClient; FIXME: Implement after MQTT lesson
+    @Resource
+    private IMqttClient client;
     @Value("false")
     private boolean isHealthy;
 
+    @Resource
+    private static GetStatusResponse latestStatus;
+    @Resource
+    private String[] message = {""};
```

Comment:
Java doesn't compile with these ``@Resources`` and therefore doesn't run
```suggestion
    private IMqttClient client;
    @Value("false")
    private boolean isHealthy;
    private GetStatusResponse latestStatus;
    private String[] message = {""};
```